test/test_main.py

```diff
@@ -0,0 +1,44 @@
+#   FirmwareUploader
+#   Copyright (c) 2021 Arduino LLC.  All right reserved.
+
+#   This library is free software; you can redistribute it and/or
+#   modify it under the terms of the GNU Lesser General Public
+#   License as published by the Free Software Foundation; either
+#   version 2.1 of the License, or (at your option) any later version.
+
+#   This library is distributed in the hope that it will be useful,
+#   but WITHOUT ANY WARRANTY; without even the implied warranty of
+#   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#   Lesser General Public License for more details.
+
+#   You should have received a copy of the GNU Lesser General Public
+#   License along with this library; if not, write to the Free Software
+#   Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+
+import json
+import semver
+
+
+def test_help(run_command):
+    result = run_command("help")
+    assert result.ok
+    assert result.stderr == ""
+    assert "Usage" in result.stdout
```

Comment:
I see no need for this test, the `help` command  is generated automatically by Cobra.

Reply:
Actually i took it from the [arduino-cli](https://github.com/arduino/arduino-cli/blob/cbe85bb5725a561f90ce79c3509ba64105a801ae/test/test_main.py#L22). Should I remove it?

Reply:
Yeah, not really necessary in the CLI either.